niryo_one_commander/scripts/robot_commander.py

```diff
@@ -94,6 +100,14 @@ def callback_robot_command(self, req):
             return self.execute_command(cmd)
         except RobotCommanderException as e:
             return self.create_response(e.status, e.message)
+        
+    def callback_set_plan_and_execute(self, req):
```

Comment:
This function is not used in the code, plus I guess that you wrote that in prevision of a service calling directly the MoveGroup to execute a trajectory. 
Could you remove this for now ? as 
1. it is not used and 
2. All commands must go through the robot_action_server to ensure validation and check for concurrent commands

Reply:
correct